src/service/video/docking.js

```diff
@@ -384,6 +404,14 @@ export class VideoDocking {
         /* opt_ext */ 'amp-video-docking');
   }
 
+  /** @private */
+  maybeInstallSlot_() {
+    if (!this.getSlot_()) {
+      return;
+    }
+    registerElement(this.ampdoc_.win, SLOT_ELEMENT_TAG, AmpVideoDockSlot);
```

Comment:
Consider moving the call to `registerElement` and the definition for `AmpVideoDockSlot` into a separate file to keep this file focused

Reply:
No more `registerElement`.